src/shared/common/firebaseFunctions.js

```diff
@@ -115,6 +115,11 @@ export const mapStateToPropsForGroups =
                     data[prefix][projectId]);
             }
         }
+
+        // Here we define a boolean value 'continueMapping' which will tell us,
+        // if there are more groups available for mapping for this user.
+        let continueMapping;
+        continueMapping = true;
```

Comment:
Any reason not to write `let continueMapping = true`?

Reply:
no particular reason. so I will adjust as you recommended